suite-native/module-connect-device/src/components/PinFromControlButtons.tsx

```diff
@@ -1,17 +1,71 @@
+import { useCallback, useEffect } from 'react';
```

Comment:
Typo in the filename. `PinFromControlButtons.tsx` --> `PinFormControlButtons.tsx`